doc/classes/ProjectSettings.xml

```diff
@@ -2311,6 +2311,9 @@
 			[b]Note:[/b] This property is only read when the project starts. To change the physics FPS at runtime, set [member Engine.physics_ticks_per_second] instead.
 			[b]Note:[/b] Only [member physics/common/max_physics_steps_per_frame] physics ticks may be simulated per rendered frame at most. If more physics ticks have to be simulated per rendered frame to keep up with rendering, the project will appear to slow down (even if [code]delta[/code] is used consistently in physics calculations). Therefore, it is recommended to also increase [member physics/common/max_physics_steps_per_frame] if increasing [member physics/common/physics_ticks_per_second] significantly above its default value.
 		</member>
+		<member name="rendering/2d/batching/item_buffer_size" type="int" setter="" getter="" default="16384">
```

Comment:
The equivalent for this in the compatibility renderer is `rendering/gl_compatibility/item_buffer_size`. I wish I had had the foresight to make the setting something backend-agnostic like the name you use here. 

After we merge this PR, we should deprecate `rendering/gl_compatibility/item_buffer_size` and use this setting for both the RD backend and the compatibility backend. I think it should be possible to stop writing `rendering/gl_compatibility/item_buffer_size` and to just read it if present, then delete it.